conf/test.config

```diff
@@ -20,7 +20,7 @@ params {
     max_time   = '6.h'
 
     // Input data
-    input = 'https://raw.githubusercontent.com/nf-core/test-datasets/cutandrun/samplesheet/test-GSE145187-small.csv'
+    input = 'https://raw.githubusercontent.com/luslab/test-datasets/cutandrun/samplesheet/test-GSE145187-small.csv'
```

Comment:
shouldn't that be on the nf-core repo?

Reply:
Yes I need to switch these over but i cant until the PR is ready to merge as all the tests will break for anyone running the current version

Reply:
Thats a good idea, I will do this for the new release